app/src/main/res/values/strings.xml

```diff
@@ -1995,6 +1995,8 @@
     <string name="year_in_review_slide_top_articles_body"><![CDATA[Here were the articles you visited the most in %1$d: %2$s]]></string>
     <string name="year_in_review_slide_top_categories_headline">Your most interesting categories</string>
     <string name="year_in_review_slide_top_categories_body"><![CDATA[Categories group articles on similar subjects together. In %1$d, you visited articles in these interesting categories multiple times: %2$s]]></string>
+    <string name="year_in_review_slide_popular_english_articles_headline">English Wikipedia\'s most popular articles</string>
+    <string name="year_in_review_slide_popular_english_articles_body"><![CDATA[When people want to learn about our world—the good, bad, weird, and wild alike—they turn to Wikipedia. The top %1$d visited articles on English Wikipedia were: %2$s]]></string>
```

Comment:
Even though this will probably only be used in English, and even though it will probably only be 5, this should be `plural`.

Reply:
Done